index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE HTML>
+<html lang="en">
+<head>
+	<title>Jordan Addis - Portfolio</title>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<!-- FavIcon -->
+	<link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png">
+	<link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png">
+	<link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png">
+	<!-- Styling -->
```

Comment:
best practice to move the styling down as the last thing mentioned after the fontawesome stuff so that it will always override other similar precendence styles

Reply:
Have moved this